datacite-data/build.gradle

```diff
@@ -0,0 +1,38 @@
+import java.nio.file.Files
+import java.nio.file.Paths
+
+plugins {
+    id 'java'
+    id "org.unbroken-dome.xjc" version "2.0.0"
```

Comment:
I still do not understand when we should call `apply plugin`. I think it should be called here, but since the code works maybe it is not necessary.

Reply:
`plugins` applies the plugin, and is used in the newer version of the DSL, however multi-project builds do not support this, so `apply plugin` is called in e.g. `allProjects` and `subProjects`. Note that applying e.g. nebula-lint in allProjects is contingent on it already having been applied in the buildSource build.gradle root.